charts/ai-software-templates/chatbot/0.1.0/templates/app-config.yaml

```diff
@@ -12,5 +12,6 @@ data:
   APP_NAMESPACE: "{{ .Values.application.namespace }}"
   GITHUB_ORG_NAME: "{{ .Values.gitops.githubOrgName }}"  
   GITHUB_SOURCE_REPO: "{{ .Values.gitops.gitSourceRepo }}"
-  GITHUB_TETKON_SOURCE_REPO: "thepetk/chatbot-test-chart-repo"
-  GITHUB_DEFAULT_BRANCH: "{{ .Values.gitops.gitDefaultBranch }}"
\ No newline at end of file
+  GITHUB_TETKON_SOURCE_REPO: "redhat-ai-dev/ai-lab-helm-charts"
```

Comment:
Since this var is tekton source repo is it this repo or the pipelines repo? just for my understanding

Reply:
Yeah originally it was meant to be the rhdh pipelines repo so it's always up-to-date. But due to the different `update-deployment` task (we now deploy directly using the docker-push pipeline) we also had to customize the pipeline.

So the script looks in the `pac` forlder of the `ai-lab-helm-charts` where the customized Pipeline exists.

Reply:
Ah okay that makes sense then thank you for explaining!